sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics/src/Customizations/Models/CollectionConfigurationError.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+
+namespace Azure.Monitor.OpenTelemetry.LiveMetrics.Models
+{
+    internal partial class CollectionConfigurationError
+    {
+        public static CollectionConfigurationError CreateError(
+            CollectionConfigurationErrorType errorType,
+            string message,
+            System.Exception? exception,
+            params Tuple<string, string>[] data)
```

Comment:
I see you're creating Tuples, and then immediately converting them to KeyValuePairString.

```
CollectionConfigurationError.CreateError(
      ...,
      Tuple.Create("MetricId", metricInfo.Id),
      Tuple.Create("TelemetryType", metricInfo.TelemetryType.ToString())));
```

```
Array.ConvertAll(data, tuple => new KeyValuePairString(tuple.Item1, tuple.Item2))
```

I think instead you could pass in individual KVPs.
`params KeyValuePairString<string, string>[]` ?

This would also avoid the Tuple allocations which are immediately thrown away.